lib/nerves_hub/scripts/runner.ex

```diff
@@ -0,0 +1,65 @@
+defmodule NervesHub.Scripts.Runner do
+  use GenServer
+
+  alias NervesHubWeb.Endpoint
+
+  defmodule State do
+    defstruct [:buffer, :from, :receive_channel, :send_channel]
+  end
+
+  def send(device, command) do
+    {:ok, pid} = start_link(device)
+    GenServer.call(pid, {:send, command.text}, 10_000)
```

Comment:
```suggestion
  def send(device, command, timeout \\ 10_000) do
    {:ok, pid} = start_link(device)
    GenServer.call(pid, {:send, command.text}, timeout)
```

Should we allow this to be configurable since scripts will be quite variable?